shell/models/management.cattle.io.roletemplate.js

```diff
@@ -56,6 +56,25 @@ export const VERBS = [
 ];
 
 export default class RoleTemplate extends SteveModel {
+  get availableActions() {
+    const out = super._availableActions;
+
+    const toFilter = ['goToEdit', 'promptRemove'];
+    const editActions = out.filter((a) => {
+      if ( toFilter.includes(a.action) ) {
+        return a;
+      }
+    });
+
+    if ( editActions ) {
```

Comment:
The `if (editActions)` will always return true because an empty array is truthy.

Can it be changed to `if (editActions.length > 0)`?

Reply:
Ah nice catch, fixed!